tests/cache/dictionary.rs

```diff
@@ -92,7 +92,7 @@ mod dictionary_get_fields {
             .dictionary_get_fields(
                 cache_name,
                 item.name(),
-                item.value().keys().cloned().collect(),
+                item.value().keys().cloned().collect::<Vec<_>>(),
```

Comment:
🤔

Reply:
yeah I think I've found a way to collect those items as `Vec<String>` instead, will update